src/pages/docs/chat/api/javascript/message.mdx

```diff
@@ -0,0 +1,393 @@
+---
+title: Message
+meta_description: ""
+---
+
+The `Message` interface represents a single message in a chat room.
+
+---
+
+## Accessors
+
+It has the following accessors:
```

Comment:
These are methods rather than accessors I would say?

Reply:
Is this a quirk of TypeDoc in how it displays them?